polkadot-launch/config.json

```diff
@@ -0,0 +1,38 @@
+{
+	"relaychain": {
+		"bin": "../../polkadot/target/release/polkadot",
+		"chain": "rococo-local",
+		"nodes": [
+			{
+				"name": "alice",
+				"wsPort": 9944,
+				"port": 30444
+			},
+			{
+				"name": "bob",
+				"wsPort": 9955,
+				"port": 30555
+			}
+		]
+	},
+	"parachains": [
+		{
+			"bin": "../target/release/canvas",
+			"id": "200",
+			"balance": "1000000000000000000000",
```

Comment:
Not sure how much sense this makes, since we use the balance of the relaychain. But the field is required and if I set it to `0` the contract deployment does no longer succeed (I think it already is stuck in submitting the transaction).

@riusricardo Do you know how the `polkadot-launch` `balance` config is set in those cases?

Reply:
I'm not sure, last time I used it I had to modify the chain spec because this number didn't have any effect on the balance.

Reply:
Ok, then maybe just leave it since it's just used for local testing anyway.